client/src/components/ContentNavigator/index.ts

```diff
@@ -247,6 +259,38 @@ class ContentNavigator implements SubscriptionProvider {
           "workbench.actions.treeView.contentdataprovider.collapseAll",
         );
       }),
+      commands.registerCommand(
+        "SAS.convertSasnbToFlw",
+        async (resource: ContentItem) => {
+          const resourceUri = getUri(resource);
+          // Ensure that the command is only available for .sasnb files
+          if (resourceUri.path.endsWith(".sasnb")) {
```

Comment:
Do we need this? My expectation is that we'll only show the context menu triggering this action when `convertSasnbToFlw` is present in the context

Reply:
If we _do_ need this, we should bail from this function early instead. Something like...

```
if (!resourceUri.path.endsWith(".sasnb")) {
    return;
}
```

...and unindent the rest

Reply:
Right, not needed anymore since the command is only available for `.sasnb` files as defined in `components/ContentNavigator/utils.ts` . Fixed in f5253ce014a5fec429456474725f717e0f2609a0